php/libraries/Module.class.inc

```diff
@@ -277,8 +284,75 @@ class Module extends \LORIS\Router\PrefixRouter implements RequestHandlerInterfa
                     )
                 )
             );
+            /* Catch more specific errors that may be thrown in the codebase.
+            * Without handling these exceptions explicitly, LORIS will display
+            * a 404 error and swallow the exception, hindering debugging efforts.
```

Comment:
I don't like this first paragraph because that's not actually how PHP handles exceptions.

(The second paragraph is fine but there's no reason to have the word "NOTE" at the front.. it's just a comment.)